pkg/nodeoperations/controller_test.go

```diff
@@ -281,7 +281,7 @@ func Test_handleNodeMaintenance(t *testing.T) {
 
 		// Expected Deployment pod was deleted from tainted node
 		err = c.client.Get(ctx, client.ObjectKey{Namespace: podcontroller.Namespace, Name: podcontroller.Name}, &podcontroller)
-		assert.True(t, k8serrors.IsNotFound(err))
+		assert.Nil(t, err)
```

Comment:
why need to change this?

Reply:
The way how k8serrors.IsNotFound works was changed and it doesn't work now it error if 'nil'. I think this change is ok as we want to delete pod without errors so just checking if error is nil is ok.

Reply:
ok. thanks.